test/system/700-play.bats

```diff
@@ -549,3 +549,18 @@ EOF
 
     run_podman kube down $yaml_source
 }
+
+@test "podman kube generate filetype" {
+    YAML=$PODMAN_TMPDIR/test.yml
+    run_podman create --pod new:pod1 --security-opt label=level:s0:c1,c2 --security-opt label=filetype:usr_t --name test1 $IMAGE true
+    run_podman kube generate pod1 -f $YAML
+    run cat $YAML
+    is "$output" ".*filetype: usr_t" "Generated YAML file should container filetype usr_t"
+    run_podman pod rm --force pod1
+
+    run_podman kube play $YAML
+    if is_selinux; then
+        run_podman inspect pod1-test1 --format "{{ .MountLabel }}"
+	is "$output" "system_u:object_r:usr_t:s0:c1,c2" "Generated container should use filetype usr_t"
+    fi
+}
```

Comment:
There's probably some cleanup missing here?

Reply:
Yes, it looks like the initial pod/container used in `kube generate` need to be removed as well.

Reply:
run_podman pod rm --force pod1 
Should remove it.